cli/Cargo.toml

```diff
@@ -6,7 +6,8 @@ authors = ["Redfire <redfire75369@hotmail.com>"]
 license = "MPL-2.0"
 
 [dependencies]
-clap = "2.33.3"
+structopt = "0.3.25"
+ctrlc = { version = "3.2.1", features = ["termination"] }
```

Comment:
`ctrlc` is not a required dependency due to `rustyline`.

Reply:
Maybe for termination feature? But I think it is justifiable, as we should not depend on dependencies of our dependencies as they might change.

Reply:
There are no dependencies on `ctrlc` in spiderfire since I switched to `rustyline`, meaning it can be removed entirely.